### ISSUE TITLE
Detail view add/edit item modal: 'Static' panel of tabs

### ISSUE THREAD

**Issue by joshxg** [openmsupply/openmsupply-client#734]
The tabbing of the 'static' panel/part of the tabs in a few of the edit item modals (The inbound shipment is the best to test with): 

![image](https://user-images.githubusercontent.com/35858975/149024000-68414b52-a624-4995-aa0b-7559b99ab6f6.png)

is a little weird.

Because this is two tables, the tab order doesn't move through the row nicely and instead does just the static area. The motivation for implementing this as two tables is to ensure the LHS columns are a static width. Because we use flexing widths, if the widths differ between columns it's likely that the 'static' columns change also.

In some ways this is not too bad, because it means when you navigate to a tab, you're able to tab just the columns of that tab. However, the initial tab is then a little odd.

So, I'm not sure. Probably better to just have a standard & predictable tab order, moving from left to right, which would mean that this should become a single table. Then, we could look at redoing the original implementation using a background with an absolute position. Some other options could be to implement a way to 'style' a column (i.e. add a background to the column) - which might be useful. Also could look at implementing a 'maxWidth' property in addition to the current `minWidth` and `width` properties

**Comment by mark-prins** [comment]
The tab issue was raised separately as #762 - though that is redundant, given this issue.
In the recent issue, this was the problem raised:

![image](https://user-images.githubusercontent.com/9192912/151872937-a2dc7ce0-4aa3-4a5c-ab6e-11f452ef24e4.png)

that the tab order was unexpected, and undesired. the preference was to tab across rows, which means implementing a single table and then doing something interesting in order to support the tabs. Wondering about creating fixed width columns, and adding the first two columns to each of the tables.

**Comment by joshxg** [comment]
Yeah, I think the `maxWidth` prop is the way to go, combined with the already implemented `minWidth` and `width` props. 

Additionally a `style` prop for the background of a column as I have seen somewhere the desire to provide some contrast between columns for quick calculating/differentiation. Though, I suspect it might be difficult to get the exact styles we have now with just styling the column.
